pkg/operator/resource/resourceapply/monitoring.go

```diff
@@ -32,7 +32,7 @@ func DeleteAlertmanager(ctx context.Context, client dynamic.Interface, recorder
 
 // ApplyPrometheus applies the Prometheus.
 func ApplyPrometheus(ctx context.Context, client dynamic.Interface, recorder events.Recorder, required *unstructured.Unstructured) (*unstructured.Unstructured, bool, error) {
```

Comment:
So your plan is to change the methods, like ApplyAlertmanager, to use ApplyUnstructuredResourceImproved, and at the same time, you would also break compatibility?

Reply:
Unfortunately so, since the `bool` value is [redundant](https://github.com/openshift/library-go/pull/1833/files#diff-c44fd9ab87f08101aadd180c482e8f18c866a305b78c7289a95a518cacdf2203R30).